### PR TITLE
Add parser sshd_test_mode

### PR DESCRIPTION
**Pull request by wushiqinlou**
### All Pull Requests:

Check all that apply:

* [x] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [ ] Is this PR to correct an issue?
* [ ] Is this PR an enhancement?

### Complete Description of Additions/Changes:

This PR is used to create a new parser for Advisor rule

### CODE REVIEW

**Review by xiangce — COMMENTED** [review]
@wushiqinlou - This new spec&parser looks good to me, as discussed, would you mind marking the existing `sshd_config` parser as `deprecated`, e.g. in `v3.6.0`?

**Comment by wushiqinlou** [comment]
@xiangce - Updated, please check

File: insights/parsers/sshd_test_mode.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        maxauthtries 6
+        ciphers aes256-gcm@openssh.com,chacha20-poly1305@openssh.com,aes256-ctr,aes128-gcm@openssh.com,aes128-ctr
+        macs hmac-sha2-256-etm@openssh.com,hmac-sha1-etm@openssh.com,umac-128-etm@openssh.com,hmac-sha2-512-etm@openssh.com,hmac-sha2-256,hmac-sha1,umac-128@openssh.com,hmac-sha2-512
+
+    Examples:
+        >>> len(sshd_test_mode)
+        10
+        >>> sshd_test_mode.get("addressfamily")
+        'any'
+        >>> sshd_test_mode.get("ciphers")
```
Comment (JoySnow) [inline insights/parsers/sshd_test_mode.py:38]: Hi @wushiqinlou , as we discussed , we can see to add a note in this parser's docstrings, to tell the parser users to take care of the data type, eg. `string` or `list`, when getting data.

Comment (wushiqinlou) [reply insights/parsers/sshd_test_mode.py:38]: @wushiqinlou - Per our discussion, I updated the value format as list for all keys, please check

**Review by JoySnow — APPROVED** [review]

File: insights/parsers/sshd_test_mode.py (hunk 1 above)
Comment (JoySnow) [reply insights/parsers/sshd_test_mode.py:38]: Thanks @wushiqinlou , the changes looks good to me. 👍

**Pull request merged**